docs/.vale/Vocab/Base/accept.txt

```diff
@@ -0,0 +1,5 @@
+Avro
```

Comment:
Need to add a lot more here over time...

Reply:
I've used these when working on parts of the developer guide over last few months:
```
boolean
API
ASCII
CLI
ConnectorFactory
ConnectorMetadata
ConnectorPageSinkProvider
ConnectorPageSourceProvider
ConnectorRecordSetProvider
ConnectorSplitManager
DNS
Guice
JDK
JVM
Kerberos
Metastore
TLS
Trino
pushdown
rowId
```

Reply:
Let me add those and maybe a few others. I think I will try to keep the list sorted..